x-pack/plugins/observability_solution/entity_manager/server/lib/entities/transform/validate_transform_ids.ts

```diff
@@ -17,9 +18,15 @@ const TRANSFORM_ID_MAX_LENGTH = 64;
 export function validateDefinitionCanCreateValidTransformIds(definition: EntityDefinition) {
   const historyTransformId = generateHistoryTransformId(definition);
   const latestTransformId = generateLatestTransformId(definition);
+  const historyBackfillTransformId = generateHistoryBackfillTransformId(definition);
```

Comment:
found out we didn't account for the history backfill id (which is the longest) when writing tests